test/DebugInfo/DebugInfo-GV-with-DIE.ll

```diff
@@ -0,0 +1,24 @@
+;; Ensure that DIExpressions are preserved in DIGlobalVariableExpressions
+;; This utilizes SPIRV DebugGlobalVariable's optional field DIExpression
+
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv -o %t.spv %t.bc --spirv-debug-info-version=nonsemantic-shader-200
+; RUN: llvm-spirv -r -o %t.rev.bc %t.spv
+; RUN: llvm-dis %t.rev.bc -o %t.rev.ll
+; RUN: FileCheck %s --input-file %t.rev.ll
+
+; CHECK: ![[#]] = !DIGlobalVariableExpression(var: ![[#GV:]], expr: !DIExpression(DW_OP_constu, 1, DW_OP_stack_value))
```

Comment:
would be nice to have SPV checks as well

Reply:
Added checks for SPV.